### DISCUSSION TITLE
Nuget package 📦 C# library

### DISCUSSION THREAD

**Discussion by johnyquest7** [microsoft/MInference#61] [Ideas]
Will there be a port for this, so that it can be used in visual studio for c#? Nuget package 📦 C# library Will be helpful

**Comment by iofu728** [comment]
Hi @johnyquest7,

Thanks for your interest in MInference.

Due to limited resources, we don't currently have plans to support C#. However, if you or anyone else is interested in porting it to C#, we would be happy to mention the related project on our homepage.
